components/org.wso2.carbon.identity.fraud.detection.sift/src/main/java/org/wso2/carbon/identity/fraud/detection/sift/Constants.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.fraud.detection.sift;
+
+/**
+ * Constants class to hold the constants used in the Sift connector.
+ */
+public class Constants {
+
+    private Constants() {
+    }
+
+    public static final String SIFT_API_URL = "https://api.sift.com/v205/events?abuse_types=account_takeover";
+    public static final String RETURN_SCORE_PARAM = "&return_score=true";
+
+    // Connector configs.
+    public static final String SIFT_ACCOUNT_ID_PROP = "sift.account.id";
+    public static final String SIFT_ACCOUNT_ID_PROP_NAME = "Account ID";
+    public static final String SIFT_ACCOUNT_ID_PROP_DESC =  "Account id of the Sift account.";
+    // __secret__ prefix is used to mark the property as confidential for UI rendering.
+    public static final String SIFT_API_KEY_PROP = "__secret__.sift.api.key";
+    public static final String SIFT_API_KEY_PROP_NAME = "API Key";
+    public static final String SIFT_API_KEY_PROP_DESC = "API key of the Sift account.";
+    public static final String CONNECTOR_NAME = "sift-configuration";
+    public static final String CONNECTOR_FRIENDLY_NAME = "Sift Configuration";
+    public static final String CONNECTOR_CATEGORY = "Other Settings";
```

Comment:
Can we get the connector category name from a defined constant, if there is already a constant for all connector categories?